### PR TITLE
fix(nms): bump yarn timeout to avoid preventable build failures

### PR DESCRIPTION
**Pull request by lucasgonze**
Signed-off-by: Lucas Gonze <lucas@gonze.com>

## Summary

Prevent needless test failures by increasing network timeout. This is an essential change on my machine.

The downside is that tests that are hung rather than simply slow will take longer to die.

## Test Plan

- yarn test
- yarn test:e2e
- Ensure CI tests pass

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: nms/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ COPY package.json yarn.lock babel.config.js ./
 
 # Install node dependencies
 ENV PUPPETEER_SKIP_DOWNLOAD "true"
-RUN yarn install --mutex network --frozen-lockfile && yarn cache clean
++RUN yarn install --mutex network --frozen-lockfile --network-timeout 100000 && yarn cache clean
```
[the remark this answers is not in the document]
Comment (lucasgonze) [reply nms/Dockerfile]: Fixed

**Comment by lucasgonze** [comment]
The NMS build failure is unrelated to the substance of this change.

`{
  "code": "ResourceNotFound",
  "message": "/@typescript-eslint/typescript-estree/-/typescript-estree-5.30.7.tgz: does not exist"
}`

File: nms/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (lucasgonze) [reply nms/Dockerfile]: Fixed

**Comment by sebathomas** [comment]
A bit confused about this change. Doesn't this just increase the timeout when installing third-party dependencies? I don't think it affects the test timeout.

**Comment by lucasgonze** [comment]
> Doesn't this just increase the timeout when installing third-party dependencies? I don't think it affects the test timeout.

e2e tests fail if there is a timeout while installing third-party dependencies.

**Comment by lucasgonze** [comment]
> e2e tests fail if there is a timeout while installing third-party dependencies.

For me this is a daily occurrence.

**Comment by sebathomas** [comment]
Thanks for the explanation.

**Review by sebathomas — APPROVED** [review]

**Pull request merged**